pkgs/applications/misc/swappy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, wayland
+, cairo
+, pango
+, gtk
+, pkgconfig
+, cmake
+, scdoc
+, libnotify
+, gio-sharp
+}:
+
+stdenv.mkDerivation rec {
+  name = "swappy-${version}";
+  version = "2020-02-26";
+
+  src = fetchFromGitHub {
+    owner = "jtheoof";
+    repo = "swappy";
+    rev = "b5cc433d75d77759cef139e0e232bde79196f886";
+    sha256 = "08r9hmhzzb5ac4g6zwm7w05n99v0hl2h0w4d8i694hg4pyjxb95y";
+  };
+
+  buildInputs = [ meson ninja pkgconfig cmake scdoc gio-sharp libnotify gtk pango cairo wayland];
```

Comment:
So upstream [said](https://github.com/jtheoof/swappy/issues/11#issuecomment-591671337): 

> gio-sharp does not exist in Arch.

Are you sure `gio-sharp` is absolutely needed? How come he hasn't said so?

Reply:
I get errors without it, e.g.:

```
../src/pixbuf.c:3:10: fatal error: gio/gunixoutputstream.h: No such file or directory
    3 | #include <gio/gunixoutputstream.h>
```

Reply:
It's part of `glib2` on Arch. Somehow it compiles fine on Arch without specifying the lib explicitly. I just pushed a fix in the `meson` build file.